src/library/trackcollectionmanager.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <QDir>
+#include <QList>
+#include <QObject>
+#include <QSet>
+
+#include "library/scanner/libraryscanner.h"
+#include "preferences/usersettings.h"
+#include "track/globaltrackcache.h"
+#include "util/db/dbconnectionpool.h"
+
+class TrackCollection;
+class ExternalTrackCollection;
+
+// Manages the internal and a varying number of external track collections.
```

Comment:
This is an odd sentence. How about
```suggestion
// Manages Mixxx's internal database of tracks as well as external track collections.
```
Is that description still accurate?